apps/meteor/client/views/marketplace/AppMenu.js

```diff
@@ -40,6 +41,9 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 	const buildExternalUrl = useEndpoint('GET', '/apps');
 	const syncApp = useEndpoint('POST', `/apps/${app.id}/sync`);
 	const uninstallApp = useEndpoint('DELETE', `/apps/${app.id}`);
+	const { data } = useIsEnterprise();
+
+	const isEnterprise = data?.isEnterprise ?? false;
```

Comment:
I Would Suggest to add in the end of the variable name `License`... to make it more clear